target_chains/ethereum/contracts/foundry.toml

```diff
@@ -1,5 +1,5 @@
 [profile.default]
-solc_version = '0.8.4'
+solc_version = '0.8.21'
```

Comment:
This change will modify our binary artifacts and requires a version bump for us.

Reply:
this is only needed for approach 1 fwiw